tests/e2e/test_zookeeper.py

```diff
@@ -51,8 +51,8 @@ def test_zookeeper_rescale(self):
 if main():
     with Module("main"):
         clickhouse_operator_spec, chi = util.install_clickhouse_and_zookeeper(
-            chi_file='configs/test-cluster-for-zookeeper.yaml',
-            chi_template_file='templates/tpl-clickhouse-latest.yaml',
+            chi_file='../configs/test-cluster-for-zookeeper.yaml',
```

Comment:
I think we need move directory `./tests/config` to `./tests/e2e/configs` and `./test/templates/` into `./tests/e2e/templates` instead of change path

Reply:
Maybe, but then we'll need to modify some scripst